tests/test_data/uaa/create_user_already_exists.json

```diff
@@ -1,5 +1,5 @@
 {
-  "user_id": "0620883d-83f4-4e3e-87a4-7e7bcec5b71a",
+  "user_id": "some.one@ons.gov.uk",
```

Comment:
Same with the other comment above (and I won't do it for each other one as it'd be repetitive).  Does the id change?  I don't think it does